src/assisted_installer_controller/assisted_installer_controller.go

```diff
@@ -147,7 +143,32 @@ func (c controller) AddRouterCAToClusterCA(wg *sync.WaitGroup) {
 		if *cluster.Status != "installed" {
 			continue
 		}
+		break
+	}
+	c.addRouterCAToClusterCA()
+	c.unpatchEtcd()
```

Comment:
I dont fee confortable that we touch etcd and exit. Maybe we should wait a bit? Verify something?

Reply:
will not help, what is happening that it takes 3-4 minutes to apply a patch. 
On the other hand, for what we will wait? if it will be broken we can do nothing

Reply:
i think we need to add openshift tests and run them after cluster installation and feel more comfortable with this change after a while